include/glow/Runtime/RuntimeTypes.h

```diff
@@ -84,14 +84,14 @@ struct DAGNode {
   RuntimeBundle runtimeBundle;
 };
 
-/// This struct contains all the created DAGNodes from the Partitioner. The
-/// contained DAGNodes can only refer to the DAGNodes from the same DAGNodeList.
-struct DAGNodeList {
-  /// The root DAGNode pointer of each graph/function.
-  std::vector<std::unique_ptr<DAGNode>> roots;
-  /// The non-root DAGNode pointers.
-  std::vector<std::unique_ptr<DAGNode>> nodes;
-};
+using rootDAGNodeTy = std::unique_ptr<DAGNode>;
+using nodesDAGNodeTy = std::vector<std::unique_ptr<DAGNode>>;
+/// The list of a pair <rootDAGNodeTy, nodesDAGNodeTy>. Each pair represents a
+/// DAG. The first one is the root and the sencond is a list of all rest of
+/// nodes in this DAG. This list contains all the created DAGNodes from the
+/// Partitioner. The contained DAGNodes can only refer to the DAGNodes from the
+/// same DAGNodeList.
+using DAGNodeList = std::vector<std::pair<rootDAGNodeTy, nodesDAGNodeTy>>;
 
```

Comment:
It would be easier to read if we did
```
struct DAG {
  std::unique_ptr<DAGNode> root;
  std::vector<std::unique_ptr<DAGNode>> nodes;
};
```
instead of all the pairs and such.
Then we can have `using DAGList = std::vector<DAG>;`